scripts/seeding/seeding_prep.py

```diff
@@ -0,0 +1,239 @@
+
+    # sim
+    if sim:
+        # 1. pool creation
+        pool_address = _pool_creation(ebtc, wbtc)
+        # 2. fake enough $steth balance
+        _provide_collateral(steth, collateral_amount_week_1)
+    else:
+        pool_address = r.uniswap.v3pool_wbtc_ebtc
+
+    pool = interface.IUniswapV3Pool(pool_address, owner=safe.account)
+
+    # 1. open cdp
+    safe.ebtc.open_cdp(collateral_amount_week_1 * 1e18, COLLATERAL_TARGET_RATIO * 1e16)
+
+    # 2. pool seeding, mint nft's. Deposit equal 1:1 (ebtc:wbtc)
+    ebtc_bal = ebtc.balanceOf(safe.account)
+    wbtc_bal = int(ebtc_bal / 10 ** (ebtc.decimals() - wbtc.decimals()))
+    C.print(f"[green]$ebtc balance to seed the pool: {ebtc_bal}[/green]")
+    C.print(f"[green]$wbtc balance to seed the pool: {wbtc_bal}[/green]")
+
+    nft_from_0925_to_099 = safe.uni_v3.mint_position(
+        pool, PRICE_0925, PRICE_099, wbtc_bal * PCT_10 * 2, 0
+    )
+    C.print(f"[green]nft_from_0925_to_099 token id: {nft_from_0925_to_099}[/green]")
+
+    nft_from_099_to_1 = safe.uni_v3.mint_position(
+        pool,
+        PRICE_099,
+        PRICE_1,
+        wbtc_bal * PCT_40,  # token0: $wbtc
+        ebtc_bal * PCT_40,  # token1: $ebtc
+    )
+    C.print(f"[green]nft_from_099_to_1 token id: {nft_from_099_to_1}[/green]")
+
+    nft_from_1_to_101 = safe.uni_v3.mint_position(
+        pool,
+        PRICE_1,
+        PRICE_101,
+        wbtc_bal * PCT_40,  # token0: $wbtc
+        ebtc_bal * PCT_40,  # token1: $ebtc
+    )
+    C.print(f"[green]nft_from_1_to_101 token id: {nft_from_1_to_101}[/green]")
+
+    nft_from_101_to_108 = safe.uni_v3.mint_position(
+        pool, PRICE_101, PRICE_108, 0, ebtc_bal * PCT_10 * 2
+    )
+    C.print(f"[green]nft_from_101_to_108 token id: {nft_from_101_to_108}[/green]")
+
+    if not sim:
+        safe.post_safe_tx()
+    else:
+        return (
+            nft_from_0925_to_099,
+            nft_from_099_to_1,
+            nft_from_1_to_101,
+            nft_from_101_to_108,
+        )
+
+
+# @note same method can be called for W3 & W4
+def seed_pool_w3(sim=False, nfts_list_sim=[]):
+    safe.init_ebtc()
+
+    # week 2 targets
+    collateral_amount_week_2 = 200
+
+    # tokens
+    wbtc = safe.contract(r.assets.wbtc)
+    ebtc = safe.contract(r.assets.ebtc)
+    steth = safe.contract(r.assets.steth)
+
+    # sim
+    if sim:
+        # 1. fake enough $steth balance
+        _provide_collateral(steth, collateral_amount_week_2)
+        nft_from_0925_to_099 = nfts_list_sim[0]
+        nft_from_099_to_1 = nfts_list_sim[1]
+        nft_from_1_to_101 = nfts_list_sim[2]
+        nft_from_101_to_108 = nfts_list_sim[3]
+    else:
+        # @note here the real nft ids should be retrieve ideal from json files once week 1 is executed!
+        nft_from_0925_to_099 = 0
+        nft_from_099_to_1 = 0
+        nft_from_1_to_101 = 0
+        nft_from_101_to_108 = 0
+
+    # 1. open cdp
+    safe.ebtc.open_cdp(collateral_amount_week_2 * 1e18, COLLATERAL_TARGET_RATIO * 1e16)
```

Comment:
I don't think the subsequent weeks should open new CDPS but rather top-up the existing one.